pennylane/interfaces/execution.py

```diff
@@ -39,8 +39,10 @@
 
 INTERFACE_MAP = {
     None: "Numpy",
+    "auto": "auto",
     "autograd": "autograd",
     "numpy": "autograd",
+    "scipy": "autograd",

```

Comment:
```suggestion
    "scipy": "Numpy",
```
Scipy wouldn't be trainable, so it should map to the non-trainable option.